test/types/record.test.ts

```diff
@@ -19,15 +19,25 @@
 
 import Record from '../../types/record'
 
+interface Person {
+  name: string
+  age: number
+}
+
 const record1 = new Record(['name', 'age'], ['Alice', 20])
 const record2 = new Record(['name', 'age'], ['Bob', 22], { key: 'value' })
```

Comment:
Could you check if we still could get the value of the key entry? 

`const record2Get1: any = record2.get('key')`

Reply:
I believe, this is checked [at lines 72-79](https://github.com/neo4j/neo4j-javascript-driver/pull/532/files#diff-90dd771e8066d69253abb1ec7d0feb0f63c21505960146852393df55e2ab2ce5R72-R79)

Reply:
It's not, i think you remove the line by accident.

Reply:
Oh, right, I get it now

Reply:
I couldn't find any usage of the third parameter of `Record` constructor (`fieldLookup`); judging from the definition, it seems to be an aliasing mechanism:

```ts
const record = new Record([ "name" ], [ "Alice" ], { firstName: "name" })

record.get("name")
// "Alice"

record.get("firstName")
// "Alice"
```

@bigmontz, Could you confirm? If so, then I should change the test cases a bit, because there's no such key here as `'value'`, there are only `"name"` and `"age"`.

Reply:
Okay, nevermind. I've done my research, the type of the `fieldLookup` parameter persists since its [definition in `1.7.7`](https://github.com/neo4j/neo4j-javascript-driver/commit/8a3dc008f5767633bee688b410aafab45527c171#diff-a300f394a24c4adc5c7700248ef506551c724ad206949915a5bbdbf5c571fc12R10). Either the functionality associated with its type has changed over time, or this was an error from the start. In either case, now `fieldLookup` is actually of type `{ [key: string]: string }`, – changing to that.

Reply:
Hi @parzhitsky, actually, i've check [here](https://github.com/neo4j/neo4j-javascript-driver/blob/79d7fd431d83e8ef11a9ad5e0b50a4731780195d/src/record.js#L56) and the def is wrong on the ts file, it works as alias but it is index based just like in our previous message.

Reply:
> now fieldLookup is actually of type `{ [key: string]: string }`, – changing to that

Sorry, meant to say `{ [key: string]: number }`

Reply:
Exactly